LEGO1/mxpalette.cpp

```diff
@@ -1,8 +1,47 @@
 #include "mxpalette.h"
 
+// OFFSET: LEGO1 0x100bee30
+MxPalette::MxPalette()
+{
+  this->m_overrideSkyColor = FALSE;
+  this->m_attached = NULL;
+  GetDefaultPalette(this->m_entries);
+  this->m_skyColor = this->m_entries[0x8d];
+  // FIXME: Incomplete
+}
+
+// OFFSET: LEGO1 100bef90
+MxPalette::~MxPalette()
+{
+  delete m_attached;  // yes this matches more
```

Comment:
You need to release COM pointers using the Release method.
```suggestion
  if (m_attached) {
    m_attached->Release();
  }
```

Reply:
In addition, this is almost certainly not `m_attached` being released here. There also is no member `m_attached`. `MxCore` is the base class, its dtor will be called implicitly.

Reply:
![image](https://github.com/isledecomp/isle/assets/24401303/ce8813fc-2050-4987-91d1-25bb371bb744)

Reply:
Addressed it here: https://github.com/ItzSwirlz/isle/pull/1